src/models/searchservice/response/directanswer/StringValueDirectAnswer.ts

```diff
@@ -9,4 +9,76 @@ import { BuiltInFieldType } from './BuiltInFieldType';
  */
 export interface TextValueDirectAnswer extends BaseFieldValueDirectAnswer<string | string[]> {
   fieldType: EnumOrLiteral<BuiltInFieldType.SingleLineText | BuiltInFieldType.MultiLineText>
+}
```

Comment:
Ooh I was ok with `StringValueDirectAnswer` since there are other "string" types in here, but I can rename!

Reply:
should the name be TextFieldValueDirectAnswer instead? or in general adding "FieldValue" to the name of these new field value DA interfaces since they don't apply to featured snippets DA?